### ISSUE TITLE
Thumbnails do not always load on item page when MediaViewer enabled

### ISSUE THREAD

**Issue by recived** [DSpace/dspace-angular#2660]
**Describe the bug**
Dspace >=7.6.0. Tested in local server,  demo.dspace.org, testrephip.unr.edu.ar.  If the item page has not been visited for a long time , then the thumbnail does not load (page needs to be refreshed or open in new window).  Full item page is ok. I suspect the problem is in the media viewer.

![image](https://github.com/DSpace/dspace-angular/assets/29452010/8d4583a5-6bbc-45cc-8577-dda5c03642a9)

**Comment by recived** [comment]
/src/app/item-page/media-viewer/media-viewer.component.html, after removing <div class="media-viewer" ***ngIf="!isLoading"**> problem disappears. Thumbnails load fine.

**Comment by tdonohue** [comment]
Pulled this over to the 7.6.x maintenance board.  Needs a volunteer to investigate & create a PR to fix.

**Comment by mspalti** [comment]
We are seeing this issue fairly consistently in 7.6.1. Adding manual change detection seems to fix the problem #2673

**Comment by recived** [comment]
> We are seeing this issue fairly consistently in 7.6.1. Adding manual change detection seems to fix the problem #2673

Thank you. problem solved.

**Comment by tdonohue** [comment]
I was able to reproduce this locally in the following scenario (Not sure if this is a common scenario, but I wanted to point it out as well):
1. Enable [MediaViewer](https://wiki.lyrasis.org/display/DSDOC7x/User+Interface+Configuration#UserInterfaceConfiguration-MediaViewerSettings)
2. Visit an Item page for an Item that has a Thumbnail from a PDF Bitstream.  Thumbnail doesn't load.
    * However, I've found when the Item has a thumbnail created from an image Bitstream, it will oftentimes load properly. (At least I've found it harder to reproduce in this scenario.)

In any case, I've found that @mspalti 's small fix in #2673 also works for me.
